### DISCUSSION TITLE
Is there unexpected overhead when configuring a wagmi app with 10+ default chains?

### DISCUSSION THREAD

**Discussion by kamwithak** [wevm/wagmi#3265] [Q&A]
Question related to https://wagmi.sh/core/chains#wagmicorechains

When invoking,

```
  const { chains, publicClient, webSocketPublicClient } = configureChains(
    defaultChains,
    providers,
  );
```

We are interested in adding around 10 default chains to our app. 

I am curious of any externalities that may arise if `defaultChains` includes 10+ chains?

**Comment by tmm** [comment]
None that I'm aware of. Feel free to follow up if you run into anything.
